FreeRTOS/FreeRTOS
# GitAction - Release Packager

Description
-----------
Add git action for automatically creating release, tag, and uploading release assets.

- `freertos_zipper.py`
    - script for pulling, pruning, and zipping FreeRTOS from main repo @somecommit
- `version_number_update.py`
    - script for updating all FreeRTOS version numbers to input arg
- `release-packager.yml`
    - git action that takes in commit ID and release version as args and proceeds to create tag, create release, and upload release assets

Test Steps
-----------
Excercise the `FreeRTOS-Release-Packager` action, verify the produced release and tag, as well as the uploaded zip

Related Issue
-----------
n/a


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
